app/controllers/posts_controller.rb

```diff
@@ -64,32 +64,26 @@ def save
     @post.update(saved: true)
     redirect_back(fallback_location: root_path)
   end
-
+  
   def unsave
     @post = Post.find(params[:id])
     @post.update(saved: false)
     redirect_back(fallback_location: root_path)
-  end
-  
+  end 
 
-  def saved_posts
-    @saved_posts=Post.where(saved: true)
-  end
   def close
-  @post = Post.find(params[:id])
-  @post.closed
-  @post.update(closed: "true")
-end
+    @post = Post.find(params[:id])
+    @post.closed
+    @post.update(closed: "true")
```

Comment:
do we need these two lines?

Reply:
Yes, we are here updating boolean value of closed to true